data/categories.txt

```diff
@@ -9164,6 +9168,7 @@ zh-Hant:^山屋
 sponsored-holiday
 en:^Christmas or New Year Celebrations
 ru:^Празднование Рождества или Нового года
+hu:^Gyűjtsd össze az összes tételt, és nyerj!
```

Comment:
Your translation seems outdated.

Reply:
​Yes, indeed. The HU translation should be:
"Karácsonyi vagy újévi ünnepség"